catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -149,6 +162,7 @@ private void createIdentityNode() throws FederationAdminException {
 
             federationAdminService.addRegistryEntry(identityMetacard);
         }
+        LOGGER.info("Successfully create registry identity node: {}", registryPackageId);
```

Comment:
s/b created 

Should this be at info level? Seems like a debug level statement